modules/release-notes/pages/sdk-release-notes.adoc

```diff
@@ -35,44 +35,22 @@ This {release} version of the software and programming language should not be us
 
 The {release} release includes the following new features and capabilities:
 
-- The Candid web interface now has a "Lucky" button that can generate random values when the input field is empty.
-- Various updates to JS userlib, including:
-* Unit is renamed to Null
-* Support Principal type
-* Support service and function references
-- Motoko now:
-* supports type argument inference for automatic instantiation of type-parametric functions;
-* adds primitive Principal and supporting (binary) Blob types
-* documents its support for caller identification (using principals) by selective parameterization of  shared functions.
-* includes actor references and shared function types as shareable types that can be transmitted across canisters
-- Native support for Principal type in Candid (IDL).
-- Improvements to language server support.
-- Subcommand `+--clean+` of `+dfx start+` can be used to clean the state of current project.
-+
-This clears checkpoints in your project and is helpful for starting your project from scratch or debugging.
-- Flags `+--v, --verbose+` and `+--q, --quiet+` for `+dfx+` can be used to toggle across various levels of information provided by messages, as desired.
-+
-These flags affect messages only and not the output or return code of the process. Using the verbose flag twice, for example, will provide `+DEBUG+` and `+TRACE+` messages, the highest level of verbosity.
+- New logging capability that supports logging informational, warning, error, debug, and trace messages to a file in addition to standard error and standard output.
+- Specifying the `--verbose` option generates logs that include debug and trace messages.
```

Comment:
1 verbose will generate debug messages, 2 verboses will generate trace as well.

The best way to look at verbose/quiet flags is increment/decrement of level; we have 6 levels (CRITICAL, ERROR, WARNING, INFO, DEBUG, TRACE). By default we log INFO and above. Adding a verbose flag will add one more level, adding a quiet flag will remove a level.

So to remove all messages, one can use `-qqqq`. To log everything, one can use `-vv`. There's a lot of TRACE so someone should be careful.

Makes sense?

Reply:
I put this info into the CLI section instead of the release notes.